.github/CODEOWNERS

```diff
@@ -165,6 +165,7 @@ packages/react-components/react-portal-compat @microsoft/teams-prg
 packages/react-components/react-alert @microsoft/teams-prg
 packages/react-components/react-portal-compat-context @microsoft/teams-prg
 #packages/react-components/theme-designer @microsoft/cxe-coastal
+#packages/react-components/global-context @microsoft/teams-prg
```

Comment:
chore: can we uncomment this pls?

Reply:
@Hotell - what is the motivation for the generator to add the entry commented out?

Reply:
ok, I see the other PR now, disregard